server/logs.lua

```diff
@@ -86,6 +85,7 @@ RegisterNetEvent('qb-log:server:CreateLog', function(name, title, color, message
         logQueue[name][#logQueue[name] + 1] = { webhook = webHook, data = embedData }
 
         if #logQueue[name] >= 10 then
+            local postData = {}
```

Comment:
Try initialising `postData` as nil, or no value.

Reply:
Seems that this change does not fix the "issue"